app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -268,8 +293,19 @@ interface AcuantCameraContextProps {
  * scope.
  */
 const getActualAcuantCameraUI = (): AcuantCameraUIInterface => {
-  if (window.AcuantCameraUI) {
-    return window.AcuantCameraUI;
+  // evaluate the arguments the function start takes
+  // if the second argument is not a function, it is the current start method, so just return the AcuantCameraUIInterface as is
+  if (window.AcuantCameraUI && typeof window.AcuantCameraUI.start.arguments[1] !== 'function') {
```

Comment:
I don't think this will work, since JS does not augment functions with argument type information like this. 

Looking at the diff between 11.8 and 11.9, it looks like they went from:

```
start(objectWithCallbacks, errorCallback, options)
```

to:

```
start(objectWithCallbacksIncludingErrorCallback, options)
```

I think you want to build a set of 3 arguments that you can pass to both versions of the code. This means:

- Put the error callback on the first argument
- Augment the error callback with options and pass that as the second argument

So something like:

```javascript
const callbacks = {
  onError() { /* ... */ },
  onCropped(response) { /* ... */ },
} 

const options = { /* ... */ }

const errorCallbackMushedTogetherWithOptions = (...args) => {
  return callbacks.onError(...args)
}
Object.keys(options).forEach(key => {
  errorCallbackMushedTogetherWithOptions[key] = options[key];
});

// Support 11.8 or 11.9
AcuantCameraUI.start(callbacks, errorCallbackMushedTogetherWithOptions, options)
```
```

Reply:
This is really helpful, thank you!  I will try this approach.